headers/functions/overlay29.h

```diff
@@ -44,6 +59,9 @@ bool MonsterIsType(struct entity* entity, enum type_id type_id);
 bool IqSkillIsEnabled(struct entity* entity, enum iq_skill_id iq_id);
 int GetMovePower(struct entity* entity, struct move* move);
 void AddExpSpecial(struct entity* attacker, struct entity* defender, int base_exp);
+uint8_t GetSleepAnimationId(struct entity* entity);
+void EndFrozenClassStatus(struct entity* user, struct entity* target, bool print_to_log);
```

Comment:
Can we just call the third param `bool log`? It lines up better with the `log_failure` params in some of the other status-related functions.